CHANGELOG.md

```diff
@@ -60,6 +61,7 @@ authentication is still required for an email notifier, but the user can now cho
   - This means the configuration for the init container has been updated to include "POSTGRES_PASSWORD_FILE" and some volume mounts which are now required.
 - Debian 9 has reached EOL, so Scanner now marks Debian 9 images as stale.
   - The Debian Security Tracker has also stopped tracking Debian 9 vulnerabilities, so there will be no more new Debian 9 vulnerabilities.
+- Support for violation comments and process comments has been removed.
```

Comment:
I guess this is a duplicate. The feature was still there in 3.71, right?

Reply:
Tags and comments are two different features. Comments features was removed in 3.71.